modules/asg/variables.tf

```
@@ -288,6 +288,12 @@ variable "security_group_ids" {
   default     = []
 }
 
+variable "tags_dest" {
```

```suggestion
variable "tag_specifications_targets" {
```
Can you please update the variable name to a more meaningful one?

---

Thanks, updated it.